src/plugins/intel_gpu/src/kernel_selector/kernels/concatenation/concatenation_kernel_simple_ref.cpp

```diff
@@ -99,6 +99,26 @@ ConcatenationKernelBase::DispatchData ConcatenationKernel_simple_Ref::SetDefault
     return dispatchData;
 }
 
+JitConstants ConcatenationKernel_simple_Ref::GetJitConstants(const concatenation_params& params) const {
+    auto jit = ConcatenationKernelBase::GetJitConstants(params);
+
+    if (!params.fused_ops.empty()) {
+        const auto& input = params.inputs[0];
+        std::vector<std::string> idx_order;
+
+        if (input.Dimentions() == 6) {
+            idx_order = { "b", "f", "w", "z", "y", "x" };
```

Comment:
That looks incorrect. Fused ops are applied to concatenation result, so you should use output indices here to have correct extra data loading

Reply:
Fixed, and for concatenation_kernel_ref too